### ISSUE TITLE
Support for table cards

### ISSUE THREAD

**Issue by jmidd** [treethought/flask-assistant#112]
Have you seen the new table cards ? https://developers.google.com/actions/assistant/responses - Any plans to support them ?
If there are no plans, could someone point me in the right direction and I'll try make the changes myself.

**Comment by treethought** [comment]
Yes, I have seen them and do plan to support them in addition to the media responses.

I will try to implement them soon, but if you'd like to look into it yourself, you would create a response class in response.py, provide methods to add rows to the table, and return the proper JSON as a response.

It would likely work similar to the ListSelector and CarouselCard classes

**Comment by jmidd** [comment]
Thanks for the reply, I'll have a look when I've got the voice part of my action finished.
